docs/ldml/tr35-personNames.md

```diff
@@ -607,7 +611,7 @@ For example:
 3. Pattern C is discarded, because it has the least number of populated name fields.
 4. Out of the remaining patterns A and B, pattern B wins, because it has only 3 unpopulated fields compared to pattern A.
 
-#### 6.5.1 <a name="process-the-namepattern" href="#process-the-namepattern">Choose a namePattern</a>
+#### 6.5.1 <a name="process-the-namepattern" href="#process-the-namepattern">Process the namePattern</a>
```

Comment:
This is a good reason to drop the section numberings, it would just become:

```markdown
#### Process the namePattern
```

Reply:
… which i plan to do soon

Reply:
Hmmm. We should discuss that first. The downside of that is that:
1. In the unprocessed view, there wouldn't be any links.
2. It would be very easy for someone to break external links, by inadvertently editing the title.

Reply:
I felt ok about changing the links here, because this was in tech preview. But once it 'gradulates'...

Reply:
Sorry, my comments were directed to "dropping the <a name=...>", not dropping the section header.

Reply:
@macchiati to discuss in CLDR-15473  

1. there would be.  consider:

```markdown
- [foo](#foo)
…
# Foo
```

![image](https://user-images.githubusercontent.com/855219/224193931-ea0b05a4-f4a7-422f-ba64-4e0ecacd0c3e.png)

The headings are their own links. Environments can navigate them appropriately without needing HTML.

2. That can already happen, but I plan to keep a stabilized set of headings and make sure they don't get removed over releases.

Reply:
that works.

Reply:
the `<a name… ` in tr35-personNames.md look like they were calculated _from_ the header names, as if someone added the section numbering and `<a name…` after the fact.  So it will be easy to move it back.  They may need to be retained on other sections for stable links.

Reply:
By the way, it's actually somewhat inverted.  Headings  such as `#### Name` are better supported, parsing the `<a name… ` chokes quite a few tools.  Markdown validation fails with "no such header: X" because the HTML isn't parsed.  `<a name` is in spec and works fine for HTML generation, but not as well supported as native markdown.

Reply:
Related (but different). We should decide this week whether we're going to remove all the section numbers in the parts or not. Don't know how much time that would take.